internal/analytics/client/async/client.go

```diff
@@ -30,9 +30,9 @@ type Client struct {
 	output           string
 	projectNameSpace string
 	eventWaitGroup   *sync.WaitGroup
-
-	sessionToken string
-	updateTag    string
+	cfg              *config.Instance
```

Comment:
We don't need this on the async client, since the async client on its own doesn't really do anything but report it through the state-svc to the sync client. Adding it on the async client would only introduce unnecessary slowdown in the code.